static/app/utils/analytics/quickStartAnalyticsEvents.tsx

```diff
@@ -12,10 +12,14 @@ export type QuickStartEventParameters = {
     todo_id: string;
     todo_title: string;
   };
+  'quick_start.task_group_completed': {
+    group: 'getting_started' | 'beyond_basics';
```

Comment:
we do not need the 'new_experience' here as it is present only in the new experience